### ISSUE TITLE
kubejs error on server script version 1.0.8

### ISSUE THREAD

**Issue by SNIPER722** [AllTheMods/ATM-8#638]
**symptoms:**
kubejs report error on OP join the word.

**invalid syntax:**
kubejs/server_scripts/tags.js extra `;` observed on line 27 and 28
```
24    'ae2:chipped_budding_quartz',
25    'ae2:flawed_budding_quartz',
26    'ae2:flawless_budding_quartz'
27  ]);
28  event.add('ars_nouveau:golem/cluster', ['minecraft:amethyst_cluster', 'ae2:quartz_cluster']);
})
```

**Comment by SNIPER722** [comment]
confirm fixed in 1.0.9 
closing

**Comment by hallool** [comment]
no still there
![grafik](https://github.com/AllTheMods/ATM-8/assets/48066058/69b5a9b9-cb5b-43e3-b6e3-2cce9a470489)
